scipy/spatial/distance.py

```diff
@@ -413,6 +422,34 @@ def sqeuclidean(u, v):
     return np.dot(u_v, u_v)
 
 
+def aitchison(u, v):
+    """
+    Computes the Aitchison distance between two 1-D arrays.
+    The Aitchison distance between 1-D arrays `u` and `v`, is defined as
+    .. math::
+       {||log \left( \frac{u}{v} \right) - mean \left(log \left( \frac{u}{v} \right) \right) ||}_2
```

Comment:
http://ns.leg.ufpr.br/lib/exe/fetch.php/pessoais:abtmartins:logratioanalysisandcompositionaldistance.pdf

Yeah, they end up being equivalent. Note that this formulation is a vectorized version of equation (1) in this reference (the equivalent equation (2) being the one in your paper).